src/Components/WebAssembly/Build/src/targets/Blazor.MonoRuntime.props

```diff
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-    <MonoLinkerI18NAssemblies>none</MonoLinkerI18NAssemblies> <!-- See Mono linker docs - allows comma-separated values from: none,all,cjk,mideast,other,rare,west -->
+    <BlazorWebAssembly18NAssemblies>none</BlazorWebAssembly18NAssemblies> <!-- See Mono linker docs - allows comma-separated values from: none,all,cjk,mideast,other,rare,west -->
```

Comment:
This shouldn't be just BlazorWebAsembly**18N**Assemblies, make sure you add the `I` too:

Reply:
This is coming from `Internationalization` = `I` + 18 letters + `N` :)